src/ensembl/src/content/app/custom-download/containers/content/preview-card/previewCardHelper.ts

```diff
@@ -0,0 +1,111 @@
+import JSONValue from 'src/shared/types/JSON';
+
+import {
+  flattenObject,
+  getProcessedAttributes
+} from 'src/content/app/custom-download/containers/content/customDownloadContentHelper';
+
+const formatResponseToArray = (responseData: any): any => {
+  const preResult: any = [];
+
+  const responseArray = flattenObject(responseData);
+  Object.keys(responseArray)
```

Comment:
`responseArray` is an object. Why is it called an array then?

Reply:
Renamed it.